src/bokeh_fastapi/__init__.py

```diff
@@ -0,0 +1 @@
+from .application import BokehFastAPI
```

Comment:
Do we really want to make this public? I would rather not have the backend completely public, but only the necessary API. Looking at the example in `test.py` the only thing we need is a function that takes the bokeh apps and optionally a FastAPI app.

Reply:
Yes, let's add some utilities and expose those.

Reply:
Will follow up on this later. We can use Quansight/ragna#416 and follow-ups as guinea pigs to see what functionality we actually need.